### DISCUSSION TITLE
Unkown command get5_listbackups

### DISCUSSION THREAD

**Discussion by jaselee0905** [PhlexPlexico/G5API#279] [Help]
When I go to select a backup to restore I get this error 
![image](https://github.com/PhlexPlexico/G5API/assets/41090702/469cfc6c-6ddd-41db-9c05-a0685bf15a1a)

![image](https://github.com/PhlexPlexico/G5API/assets/41090702/3eef9b28-17d6-4093-9661-cb7f76c5d55c)

Everything else seems to work fine

**Comment by PhlexPlexico** [comment]
If you're using MatchZy, please scroll to the bottom of this page and review `Current Limitations with Get5 Integration`

https://shobhit-pathak.github.io/MatchZy/get5/

**Reply by jaselee0905** [reply]
does it work with pugsharp? Sorry if this is the wrong place to ask I just couldn't find anything in their documentation about it

**Reply by PhlexPlexico** [reply]
I'm not sure, you would have to open up an issue with them to check and see, or ask in their respective discord :)
